code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -296,12 +296,18 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 	out += "[tail_spacing ? "<br>" : ""]<b>Hair</b><br>"
 	if(has_flag(mob_species, HAS_HAIR_COLOR))
 		out += "<a href='?src=\ref[src];hair_color=1'>Change Color</a> [HTML_RECT(rgb(pref.r_hair, pref.g_hair, pref.b_hair))] "
-	out += " Style: <a href='?src=\ref[src];hair_style=1'>[pref.h_style]</a><br>"
+	out += " Style: " \
+		+ "<a href='?src=\ref[src];previous_hair_style=1'> < </a>" \
+		+ "<a href='?src=\ref[src];next_hair_style=1'> > </a>" \
+		+ "<a href='?src=\ref[src];hair_style=1'>[pref.h_style]</a><br>"
```

Comment:
```suggestion
	out += " Style: " \
		+ "<a href='?src=\ref[src];previous_hair_style=1'> < </a>" \
		+ "<a href='?src=\ref[src];hair_style=1'>[pref.h_style]</a><br>" \
		+ "<a href='?src=\ref[src];next_hair_style=1'> > </a>"
```

Reply:
Imho, this option is inferior to mine in convenience, because due to the change of hairstyles, the middle button will constantly change in length and shift the position of the right button.